src/styles/main.scss

```diff
@@ -0,0 +1,79 @@
+$width-day: 100px;
+$gap: 1px;
+$start-day: (
+  mon 0,
+  tue ($width-day) + $gap,
+  wed ($width-day * 2) + ($gap * 2),
+  thu ($width-day * 3) + ($gap * 3),
+  fri ($width-day * 4) + ($gap * 4),
+  sat ($width-day * 5) + ($gap * 5),
+  sun ($width-day * 6) + ($gap * 6)
+);
+
+body {
+  height: 100%;
+  margin: 0;
+}
+
+.main {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: ($width-day * 7) + ($gap * 6);
```

Comment:
The width calculation for the `.calendar` class should include the padding as well. According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), when calculating widths with padding, you should include the padding in the calculation to avoid unexpected overflow or layout issues.